test/powershell/Modules/Microsoft.PowerShell.Utility/RunspaceCmdlets.Tests.ps1

```diff
@@ -0,0 +1,39 @@
+Describe "Get-Runspace cmdlet tests" -Tag "CI" {
+    BeforeAll {
+        # Clear out all unexpected runspaces
+        # id 1 is the one we started with
+        Get-Runspace |?{$_.id -ne 1} | %{$_.dispose()}
+        $CurrentRunspace = $ExecutionContext.Host.Runspace
```

Comment:
This seems unwise - e.g. what if Pester used other runspaces, or we started running tests in parallel.